Assignment/solution_student/index.html

```diff
@@ -1,35 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
 
-Sample Content
-——-
+  <head>
 
-ABOUT ME PAGE
+    <meta charset="UTF-8">
+    <title>Joe Johnson</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic|Lora:400,400italic,700,700italic' rel='stylesheet' type='text/css'>
+  
+  </head>
+  
+  <header>
 
-I ENJOY LIFE AS A DEVELOPER
+    <div class="border"></div>
 
-I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.
+    <nav>
 
+      <a href="index.html">About Me</a> / <a href="portfolio.html">Portfolio</a>
+      <h1>Joe Johnson</h1>
```

Comment:
Not a huge deal, but semantically the `<nav>` element is reserved for navigational links, so it's probably best to move the `<h1>` outside of the nav.